### PR TITLE
NMS-13864: to be remove

### PR DESCRIPTION
**Pull request by rabzobi**
### External References

    * JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13864

### CODE REVIEW

File: tools/packages/minion/minion.spec [hunk 1]
```diff
@@ -83,7 +83,7 @@ Prefix:         %{minioninstprefix}
 OpenNMS Minion is a container infrastructure for distributed, scalable network
 management and monitoring.
 
-http://www.opennms.org/wiki/Minion
+http://docs.opennms.com/horizon/latest/deployment/minion/introduction.html
```
Comment (mmahacek) [inline tools/packages/minion/minion.spec:86]:
```suggestion
https://docs.opennms.com/horizon/latest/deployment/minion/introduction.html
```

File: opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/MonitoringLocationRestService.java [hunk 2]
```diff
@@ -88,10 +88,10 @@ protected Class<OnmsMonitoringLocation> getQueryBeanClass() {
 
     @Override
     protected CriteriaBuilder getCriteriaBuilder(UriInfo uriInfo) {
-        final CriteriaBuilder builder = new CriteriaBuilder(OnmsMonitoringLocation.class);
+        CriteriaBuilder builder = new CriteriaBuilder(OnmsMonitoringLocation.class);
 
-        // Order by location name by default
-        builder.orderBy("locationName").asc();
+        // Order by location name by default and remove limit on list
+        builder = builder.orderBy("locationName").asc().limit(0);
```
Comment (j-white) [inline opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/MonitoringLocationRestService.java:94]: Unrelated changes, should be removed from this PR.

Comment (rabzobi) [reply opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/MonitoringLocationRestService.java]: I have rolled theses changes back, my apologies

**Review by mmahacek — DISMISSED** [review]
LGTM
